.pre-commit-config.yaml

```diff
@@ -1,14 +0,0 @@
-
```

Comment:
I think we should leave pre-commit in- I think it's a good defensive move for the codebase

Reply:
Offline: leaving out for now. could bring back if we set it to just run `tox -e check`